src/msw/spinctrl.cpp

```diff
@@ -352,33 +371,32 @@ bool wxSpinCtrl::Create(wxWindow *parent,
     // windows are created and the text one is set up as buddy because
     // UDM_SETBUDDY changes its size using some unknown algorithm, so setting
     // the sizes earlier is useless.
-    const int bestSpinWidth = wxSpinButton::DoGetBestSize().x;
-    const int effectiveSpinWidth = bestSpinWidth - GetOverlap();
```

Comment:
I'm suspicious of this line disappearing, it looks like we don't take the overlap into account at all any more. Why?

Reply:
Before my changes `effectiveSpinWidth` was used to calculate the width from the fixed value (`DEFAULT_ITEM_WIDTH`).
```
sizeCtrl.x = FromDIP(DEFAULT_ITEM_WIDTH) + effectiveSpinWidth;
```
But now we use our realization of `DoGetBestSize` (not `wxSpinButton::DoGetBestSize` as before), which uses `DoGetSizeFromTextSize`.